### PR TITLE
Complete Spanish language version

### PR DESCRIPTION
**Pull request by nomadscientist**
However, it has been transferred across a few formats, so I'm not sure if it will work or not...

## What did you do? 
- We finished the template for Spanish language Galaxy instance
- 

## Why did you make this change?
- Spanish language was unfinished

## How to test the changes? 
(select the most appropriate option; if the latter, provide steps for testing below)
- [ ] I've included appropriate [automated tests](https://docs.galaxyproject.org/en/latest/dev/writing_tests.html).
- [ ] This is a refactoring of components with existing test coverage.
- [ ] Instructions for manual testing are as follows:
  1. [add testing steps and prerequisites here if you didn't write automated tests covering all your changes]

I don't know how to test it.

## License
- [x] I agree to license these contributions under [Galaxy's current license](https://github.com/galaxyproject/galaxy/blob/dev/LICENSE.txt).
- [x] I agree to allow the Galaxy committers to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT). If this condition is an issue, uncheck and just let us know why with an e-mail to galaxy-committers@lists.galaxyproject.org.

## For UI Components
- [ ] I've included a screenshot of the changes

### CODE REVIEW

**Comment by dannon** [comment]
@nomadscientist Wow, thanks!  I can run the format and push it to this branch to make lint pass again.  I can't verify any of the translations, but I'll make sure it at least works :)

File: client/src/nls/es/locale.js [hunk 1]
```diff
@@ -1,206 +1,630 @@
 /** en/main localization hash - for use with requirejs' i18n plugin */
+
 define({
     // ----------------------------------------------------------------------------- masthead
+
     "Analyze Data": "Analizar datos",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
    "Analyze Data": "Analizar Datos",
```

File: client/src/nls/es/locale.js [hunk 2]
```diff
@@ -1,206 +1,630 @@
...
+
     Workflow: "Flujo de trabajo",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
    Workflow: "Flujo de Trabajo",
```

File: client/src/nls/es/locale.js [hunk 3]
```diff
@@ -1,206 +1,630 @@
...
+
     "Shared Data": "Datos compartidos",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
    "Shared Data": "Datos Compartidos",
```

File: client/src/nls/es/locale.js [hunk 4]
```diff
@@ -1,206 +1,630 @@
...
+
     "Data Libraries": "Bibliotecas de datos",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
    "Data Libraries": "Bibliotecas de Datos",
```

File: client/src/nls/es/locale.js [hunk 5]
```diff
@@ -1,206 +1,630 @@
...
-    Histories: "Historias",
+
+    Histories: "Historiales",
+
     Workflows: "Flujos de trabajo",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
    Workflows: "Flujos de Trabajo",
```

File: client/src/nls/es/locale.js [hunk 6]
```diff
@@ -1,206 +1,630 @@
...
+
     Visualizations: "Visualizaciones",
+
     Pages: "Páginas",
+
     Visualization: "Visualización",
+
     "New Track Browser": "Nuevo navegador de pistas",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
    "New Track Browser": "Nuevo Navegador de Pistas",
```

File: client/src/nls/es/locale.js [hunk 7]
```diff
@@ -1,206 +1,630 @@
...
+
     "Saved Visualizations": "Visualizaciones Guardadas",
+
     "Interactive Environments": "Entornos Interactivos",
+
     Admin: "Administración",
+
     Help: "Ayuda",
+
     Support: "Apoyo",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
    Support: "Soporte",
```

File: client/src/nls/es/locale.js [hunk 8]
```diff
@@ -1,206 +1,630 @@
...
+
     Search: "Buscar",
+
     "Mailing Lists": "Listas de correo",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
    "Mailing Lists": "Listas de Correo",
```

File: client/src/nls/es/locale.js [hunk 9]
```diff
@@ -1,206 +1,630 @@
...
+
     Videos: "Videos",
+
     Wiki: "Wiki",
+
     "How to Cite Galaxy": "Cómo citar a Galaxy",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
    "How to Cite Galaxy": "Cómo citar Galaxy",
```

File: client/src/nls/es/locale.js [hunk 10]
```diff
@@ -1,206 +1,630 @@
...
-    "Interactive Tours": "recorridos interactivos",
+
+    "Interactive Tours": "Recorridos interactivos",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
    "Interactive Tours": "Tours Interactivos",
```

File: client/src/nls/es/locale.js [hunk 11]
```diff
@@ -1,206 +1,630 @@
...
     "Login or Register": "Iniciar sesión o Registrarse",
-    "Logged in as": "Conectado como",
+
+    "Logged in as": "Se ha conectado como",
+
     Preferences: "Preferencias",
+
     "Custom Builds": "Construcciones personalizadas",
+
     Logout: "Cerrar sesión",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
    Logout: "Cerrar Sesión",
```

File: client/src/nls/es/locale.js [hunk 12]
```diff
@@ -1,206 +1,630 @@
...
-    "Saved Histories": "Historias Guardadas",
+
+    "Saved Histories": "Historiales guardados",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
    "Saved Histories": "Historiales Guardados",
```

File: client/src/nls/es/locale.js [hunk 13]
```diff
@@ -1,206 +1,630 @@
...
+
     "Saved Datasets": "Conjuntos de datos guardados",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
    "Saved Datasets": "Conjuntos de Datos Guardados",
```

File: client/src/nls/es/locale.js [hunk 14]
```diff
@@ -1,206 +1,630 @@
...
+
     "Saved Pages": "Páginas guardadas",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
    "Saved Pages": "Páginas Guardadas",
```

File: client/src/nls/es/locale.js [hunk 15]
```diff
@@ -1,206 +1,630 @@
...
+
     //Tooltip
+
     "Account and saved data": "Cuenta y datos guardados",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
    "Account and saved data": "Cuenta y Datos Guardados",
```

File: client/src/nls/es/locale.js [hunk 16]
```diff
@@ -1,206 +1,630 @@
...
+
     "Account registration or login": "Registro de cuenta o inicio de sesión",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
    "Account registration or login": "Crear Cuenta o Iniciar Sesión",
```

File: client/src/nls/es/locale.js [hunk 17]
```diff
@@ -1,206 +1,630 @@
...
+
     "Support, contact, and community": "Apoyo, contacto y comunidad",
-    "Administer this Galaxy": "Administrar esta galaxia",
+
+    "Administer this Galaxy": "Administrar este Galaxy",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
    "Administer this Galaxy": "Administrar esta instancia de Galaxy",
```

File: client/src/nls/es/locale.js [hunk 18]
```diff
@@ -1,206 +1,630 @@
...
+
     "Visualize datasets": "Visualizar conjuntos de datos",
+
     "Access published resources": "Acceso a los recursos publicados",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
    "Access published resources": "Acceder a los recursos publicados",
```

File: client/src/nls/es/locale.js [hunk 19]
```diff
@@ -1,206 +1,630 @@
...
-    "Chain tools into workflows": "Herramientas de cadena en flujos de trabajo",
+
+    "Chain tools into workflows": "Cadena de  herramientas en flujos de trabajo",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
    "Chain tools into workflows": "Encadenar herramientas en flujos de trabajo",
```

File: client/src/nls/es/locale.js [hunk 20]
```diff
@@ -1,206 +1,630 @@
...
-    "Current History": "Historia actual",
+
+    "Histories Shared with Me": "Historiales compartidos conmigo",
+
+    "Current History": "Historial actual",
+
     "Create New": "Crear nuevo",
-    "Copy History": "Historial de Copias",
+
+    "Copy History": "Copiar historial",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
    "Copy History": "Copiar Historial",
```

File: client/src/nls/es/locale.js [hunk 21]
```diff
@@ -1,206 +1,630 @@
...
+
     "Share or Publish": "Compartir o publicar",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
    "Share or Publish": "Compartir o Publicar",
```

File: client/src/nls/es/locale.js [hunk 22]
```diff
@@ -1,206 +1,630 @@
...
+
     "Show Structure": "Mostrar Estructura",
+
     "Extract Workflow": "Extraer flujo de trabajo",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
    "Extract Workflow": "Extraer Flujo de Trabajo",
```

File: client/src/nls/es/locale.js [hunk 23]
```diff
@@ -1,206 +1,630 @@
...
+
     // Delete is defined elsewhere, but is also in this menu.
+
     // "Delete":
+
     //     false,
+
     "Delete Permanently": "Borrar permanentemente",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
    "Delete Permanently": "Borrar Permanentemente",
```

File: client/src/nls/es/locale.js [hunk 24]
```diff
@@ -1,206 +1,630 @@
...
+
     "Dataset Actions": "Acciones de conjuntos de datos",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
    "Dataset Actions": "Acciones de Conjuntos de Datos",
```

File: client/src/nls/es/locale.js [hunk 25]
```diff
@@ -1,206 +1,630 @@
...
-    "Copy Datasets": "Conjuntos de datos de copia",
+
+    "Copy Datasets": "Copiar conjuntos de datos",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
    "Copy Datasets": "Copiar Conjuntos de Datos",
```

File: client/src/nls/es/locale.js [hunk 26]
```diff
@@ -1,206 +1,630 @@
...
+
     "Dataset Security": "Seguridad del conjunto de datos",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
    "Dataset Security": "Seguridad del Conjunto de Datos",
```

File: client/src/nls/es/locale.js [hunk 27]
```diff
@@ -1,206 +1,630 @@
...
+
     "Resume Paused Jobs": "Reanudar trabajos en pausa",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
    "Resume Paused Jobs": "Reanudar Trabajos en Pausa",
```

File: client/src/nls/es/locale.js [hunk 28]
```diff
@@ -1,206 +1,630 @@
...
-    "Collapse Expanded Datasets": "Contraer conjuntos de datos expandidos",
+
+    "Collapse Expanded Datasets": "Colapsar conjuntos de datos expandidos",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
    "Collapse Expanded Datasets": "Contraer Conjuntos de Datos Expandidos",
```

File: client/src/nls/es/locale.js [hunk 29]
```diff
@@ -1,206 +1,630 @@
...
+
     "Unhide Hidden Datasets": "Mostrar conjuntos de datos ocultos",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
    "Unhide Hidden Datasets": "Mostrar Conjuntos de Datos Ocultos",
```

File: client/src/nls/es/locale.js [hunk 30]
```diff
@@ -1,206 +1,630 @@
...
+
     "Delete Hidden Datasets": "Eliminar conjuntos de datos ocultos",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
    "Delete Hidden Datasets": "Eliminar Conjuntos de Datos Ocultos",
```

File: client/src/nls/es/locale.js [hunk 31]
```diff
@@ -1,206 +1,630 @@
...
-    "Purge Deleted Datasets": "Eliminar los conjuntos de datos eliminados",
+
+    "Purge Deleted Datasets": "Depurar los conjuntos de datos eliminados",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
    "Purge Deleted Datasets": "Eliminar Definitivamente los Conjuntos de Datos Eliminados",
```

File: client/src/nls/es/locale.js [hunk 32]
```diff
@@ -1,206 +1,630 @@
...
+
     Downloads: "Descargas",
+
     "Export Tool Citations": "Citas de herramientas de exportación",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
    "Export Tool Citations": "Exportar las Citas de la Herramienta",
```

File: client/src/nls/es/locale.js [hunk 33]
```diff
@@ -1,206 +1,630 @@
...
+
     "Export History to File": "Exportar historial a archivo",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
    "Export History to File": "Exportar Historial a un Archivo",
```

File: client/src/nls/es/locale.js [hunk 34]
```diff
@@ -1,206 +1,630 @@
...
+
     "Other Actions": "Otras acciones",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
    "Other Actions": "Otras Acciones",
```

File: client/src/nls/es/locale.js [hunk 35]
```diff
@@ -1,206 +1,630 @@
...
+
     "Import from File": "Importar desde archivo",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
    "Import from File": "Importar desde Archivo",
```

File: client/src/nls/es/locale.js [hunk 36]
```diff
@@ -1,206 +1,630 @@
...
+    "This history is empty": "Este historial está vacío",
+
     "No matching datasets found": "No se encontraron conjuntos de datos coincidentes",
+
     "An error occurred while getting updates from the server":
         "Se ha producido un error al obtener actualizaciones desde el servidor",
+
     "Please contact a Galaxy administrator if the problem persists":
-        "Póngase en contacto con un administrador de Galaxy si el problema persiste",
+        "Favor de contactar a un administrador de Galaxy si el problema persiste",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
        "Por favor, contacte a un administrador de Galaxy si el problema persiste",
```

File: client/src/nls/es/locale.js [hunk 37]
```diff
@@ -1,206 +1,630 @@
...
+
     //TODO:
+
     //"An error was encountered while <% where %>" :
+
     //false,
+
     "search datasets": "Conjuntos de datos de búsqueda",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
    "search datasets": "buscar conjuntos de datos",
```

File: client/src/nls/es/locale.js [hunk 38]
```diff
@@ -1,206 +1,630 @@
...
-    "You are currently viewing a deleted history!": "¡Actualmente estás viendo un historial eliminado!",
-    "You are over your disk quota": "Has sobrepasado tu cuota de disco",
+
+    "You are currently viewing a deleted history!": "¡Te encuentras visualizando un historial eliminado!",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
    "You are currently viewing a deleted history!": "¡Estás viendo un historial eliminado!",
```

File: client/src/nls/es/locale.js [hunk 39]
```diff
@@ -1,206 +1,630 @@
...
     //"Include Hidden Datasets" :
+
     //false,
 
     // ---------------------------------------------------------------------------- datasets
+
     // ---- hda-model
-    "Unable to purge dataset": "No se puede purgar el dataset",
+
+    "Unable to purge dataset": "No se puede depurar el conjunto de datos",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
    "Unable to purge dataset": "No se puede eliminar permanentemente el conjunto de datos",
```

File: client/src/nls/es/locale.js [hunk 40]
```diff
@@ -1,206 +1,630 @@
...
     // ---- hda-base
+
     // display button
-    "Cannot display datasets removed from disk": "No se pueden mostrar conjuntos de datos eliminados del disco",
+
+    "Cannot display datasets removed from disk":
+        "No se pueden mostrar conjuntos de datos que han sido eliminados del disco",
+
     "This dataset must finish uploading before it can be viewed":
         "Este conjunto de datos debe terminar de cargar antes de que se pueda ver",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
        "Este conjunto de datos debe terminar de cargar antes de visualizarlo",
```

File: client/src/nls/es/locale.js [hunk 41]
```diff
@@ -1,206 +1,630 @@
...
+
     "There was an error getting the data for this dataset":
-        "Hubo un error al obtener los datos de este conjunto de datos",
+        "Hubo un error al extraer el contenido de este conjunto de datos",
+
     // purged'd/del'd msg
+
     "This dataset has been deleted and removed from disk":
-        "Este conjunto de datos ha sido eliminado y eliminado del disco",
+        "Este conjunto de datos ha sido eliminado y removido de disco",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
        "Este conjunto de datos ha sido eliminado permanentemente del disco",
```

File: client/src/nls/es/locale.js [hunk 42]
```diff
@@ -1,206 +1,630 @@
...
+
     "This dataset has been deleted": "Este conjunto de datos ha sido eliminado",
-    "This dataset has been hidden": "Este conjunto de datos se ha ocultado",
+
+    "This dataset has been hidden": "Este conjunto de datos ha sido ocultado",
+
     format: "Formato",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
    format: "formato",
```

File: client/src/nls/es/locale.js [hunk 43]
```diff
@@ -1,206 +1,630 @@
...
+
     "Permanently remove it from disk": "Eliminar permanentemente del disco",
-    "Unhide it": "Mostrarlo",
-    "You may be able to": "Usted puede ser capaz de",
+
+    "Unhide it": "Mostrar",
+
+    "You may be able to": "Podrás tener la capacidad de",
+
     "set it manually or retry auto-detection": "Configurar manualmente o volver a intentar la detección automática",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
    "set it manually or retry auto-detection": "configurar manualmente o volver a intentar la detección automática",
```

File: client/src/nls/es/locale.js [hunk 44]
```diff
@@ -1,206 +1,630 @@
...
+
+    "Undelete it": "Recuperar",
+
     "Permanently remove it from disk": "Eliminar permanentemente del disco",
-    "Unhide it": "Mostrarlo",
-    "You may be able to": "Usted puede ser capaz de",
+
+    "Unhide it": "Mostrar",
+
+    "You may be able to": "Podrás tener la capacidad de",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
    "You may be able to": "Podrás",
```

File: client/src/nls/es/locale.js [hunk 45]
```diff
@@ -1,206 +1,630 @@
...
+
+    Tags: "Etiquetas",
+
     "Edit annotation": "Editar anotación",
+
+    "Tool Help": "Ayuda de herramienta",
+
+    // ---------------------------------------------------------------------------- admin
+    "Reset passwords": "Restablecer contraseña",
+    "Search Tool Shed": "Buscar Tool Shed",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
    "Search Tool Shed": "Buscar en el Tool Shed",
```

File: client/src/nls/es/locale.js [hunk 46]
```diff
@@ -1,206 +1,630 @@
...
+    Permissions: false,
+    Datatypes: false,
+    Convert: false,
+    Attributes: false,
+    // ---------------------------------------------------------------------------- library-dataset-view
+    "Import into History": "Importar al historial",
+    // ---------------------------------------------------------------------------- library-foldertoolbar-view
+    "Location Details": "Detalles de ubicación",
+    "Deleting selected items": "Eliminar elementos seleccionados",
+    "Please select folders or files": "Favor de seleccionar directorios o archivos",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
    "Please select folders or files": "Por favor, seleccione directorios o archivos",
```

File: client/src/nls/es/locale.js [hunk 47]
```diff
@@ -1,206 +1,630 @@
...
+    "Please enter paths to import": "Favor de ingresar rutas para importar",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
    "Please enter paths to import": "Por favor, introduce las rutas a importar",
```

File: client/src/nls/es/locale.js [hunk 48]
```diff
@@ -1,206 +1,630 @@
...
+    "Adding datasets from your history": "Agregar conjuntos de datos desde tu historial",
+    "Create New Folder": "Crear nuevo directorio",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
    "Create New Folder": "Crear Nuevo Directorio",
```

File: client/src/nls/es/locale.js [hunk 49]
```diff
@@ -1,206 +1,630 @@
...
+    // ---------------------------------------------------------------------------- library-librarytoolbar-view
+    "Create New Library": "Crear nueva biblioteca",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
    "Create New Library": "Crear Nueva Biblioteca",
```

File: client/src/nls/es/locale.js [hunk 50]
```diff
@@ -1,206 +1,630 @@
...
+    // ---------------------------------------------------------------------------- tours
+    Tours: false,
+    // ---------------------------------------------------------------------------- user-preferences
+    "Click here to sign out of all sessions.": "Haz clic aquí para salir de todas las sesiones",
+    "Add or remove custom builds using history datasets.":
+        "Agregar o eliminar construcciones personalizadas usando conjuntos de datos del historial",
+    "Associate OpenIDs with your account.": "Asociar OpenIDs a su cuenta",
+    "Customize your Toolbox by displaying or omitting sets of Tools.":
+        "Personalizar Toolbox para mostrar u omitir conjuntos de herramientas",
+    "Access your current API key or create a new one.": "Acceder a llave API actual o crear una nueva",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
    "Access your current API key or create a new one.": "Acceder a su clave API actual o crear una nueva",
```

File: client/src/nls/es/locale.js [hunk 51]
```diff
@@ -1,206 +1,630 @@
...
+    "Allows you to change your login credentials.": "Permitir modificar tus credenciales de inicio de sesión ",
+    "User Preferences": "Preferencias de usuario",
+    "Sign out": "Cerrar sesión",
+    "Manage custom builds": "Administrar construcciones personalizadas",
+    "Manage OpenIDs": "Administrar OpenIDs",
+    "Manage Toolbox filters": "Administrar filtros de Toolbox",
+    "Manage API key": "Administrar llave API",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
    "Manage API key": "Administrar clave API",
```

File: client/src/nls/es/locale.js [hunk 52]
```diff
@@ -1,206 +1,630 @@
...
+    "Change password": "Cambiar contraseña",
+    "Manage information": "Administrar información",
+    // ---------------------------------------------------------------------------- shed-list-view
+    "Configured Tool Sheds": "Tool Sheds configurados",
+    // ---------------------------------------------------------------------------- repository-queue-view
+    "Repository Queue": "Cola de repositorio",
+    // ---------------------------------------------------------------------------- repo-status-view
+    "Repository Status": "Estado del repositorio",
+    // ---------------------------------------------------------------------------- workflows-view
+    "Workflows Missing Tools": "Flujos de trabajo con herramientas faltantes",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
    "Workflows Missing Tools": "Flujos de trabajo a los que les faltan herramientas",
```

File: client/src/nls/es/locale.js [hunk 53]
```diff
@@ -1,206 +1,630 @@
...
+    "Tool Management": "Gestión de herramienta",
+    Forms: false,
+    "User Management": "Gestión de usuario",
+    "Manage jobs": "Administrar trabajos",
+    "Display applications": "Mostrar aplicaciones",
+    "Data tables": "Tablas de datos",
+    "Data types": "Tipos de datos",
+    Server: false,
+    // ---------------------------------------------------------------------------- circster
+    "Could Not Save": "No fue posible guardar",
```
Comment (beatrizserrano) [inline client/src/nls/es/locale.js]:
```suggestion
    "Could Not Save": "No se pudo guardar",
```

**Review by beatrizserrano — COMMENTED** [review]
This is great @nomadscientist! I've made some suggestions mostly for the capitalisation, but feel free to ignore them if they don't fit :)

File: client/src/nls/es/locale.js (hunk 52 above)
Comment (beatrizserrano) [reply client/src/nls/es/locale.js]: Not super sure of this... I guess it will list the workflows that are using tools not available on the server?

Comment (dannon) [reply client/src/nls/es/locale.js]: It actually might not be used anymore, but that's the original intent.  It doesn't hurt to have the translation either way :)

**Comment by nomadscientist** [comment]
Cool stuff @beatrizserrano @dannon !!

**Comment by nomadscientist** [comment]
What stands between this and merging?

**Comment by dannon** [comment]
@nomadscientist Nothing!  I was just waiting on the relevant tests to pass.  Thanks for the contribution :)

**Comment by nomadscientist** [comment]
Hey folks - there is a lot in the main galaxy pages that don't translate (for instance, 
Screenshot 2021-05-14 at 13 32 54

**Pull request merged**